Source/m_menu.h

```diff
@@ -113,6 +113,7 @@ extern int warning_about_changes, print_warning_about_changes;
 #define S_BADVID  0x400000  // killough 12/98: video mode change error
 #define S_CHOICE  0x800000  // [FG] selection of choices
 #define S_DISABLE 0x1000000 // Disable item
+#define S_NOTWRN  0x2000000 // Don't warn about change
```

Comment:
How about `S_COSMETIC`?

Reply:
Done.
I'm thinking, what if rearrange menus a bit - introduce "Cosmetic" groups since most of our additions don't change gameplay.

Reply:
Thanks. I'd rather prefer not to rearrange the menu, especially not the items that were already there in original MBF. I don't say that they were arranged perfectly in MBF, but the port's design goal still remains to be as faithful to it as possible.